src/elements/kw-share-card/kw-share-card.html

```diff
@@ -124,7 +124,7 @@
             }
         </style>
         <kw-share-shell id="shell" share="[[share]]" tombstone$="[[!share]]">
-            <div slot="share-card">
+            <div slot="share-slot">
                 <iron-image id="cover" class="cover" src="[[share.cover_url]]" sizing="cover" preload fade></iron-image>
```

Comment:
@paulvarache this is share-card, I think on share detail you have implemented the app content loading, if not implemented yet, I would suggest move the app into next ticket